common/utils.py

```diff
@@ -271,6 +271,12 @@ def _get_vcpus_by_instance_type(instances, instance_type):
         raise CriticalError(error_msg)
 
 
-def get_compute_instance_type(region, proxy_config, stack_name):
-    parameters = get_cloudformation_stack_parameters(region, proxy_config, stack_name)
-    return parameters["ComputeInstanceType"]
+@retry(stop_max_attempt_number=3, wait_fixed=5000)
+def get_compute_instance_type(region, proxy_config, stack_name, fallback):
+    try:
+        parameters = get_cloudformation_stack_parameters(region, proxy_config, stack_name)
+        return parameters["ComputeInstanceType"]
+    except Exception:
+        if fallback:
```

Comment:
I'd add some log errors in this case, to have evidence that we are using a fallback value and we are not able to get the value from the stack.

Reply:
I just realized we are already logging in the `get_cloudformation_stack_parameters` function 👍